### PR TITLE
Queue by stack implementation

### PR DESCRIPTION
**Pull request by achalesh27022003**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [ ] I've read the contribution guidelines.
- [ ] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#1999

### CODE REVIEW

File: C-Plus-Plus/sort/selection_sort.cpp [hunk 1]
```diff
@@ -1,43 +0,0 @@
...
-   int i, j, imin;
-   for(i = 0; i<size-1; i++) {
-      imin = i;   //get index of minimum data
-      for(j = i+1; j<size; j++)
-         if(array[j] < array[imin])
-            imin = j;
-         //placing in correct position
-         swap(array[i], array[imin]);
-   }
-}
```
Comment (HarshCasper) [inline C-Plus-Plus/sort/selection_sort.cpp]: Why deleting the selection sort file?

**Review by HarshCasper — COMMENTED** [review]
The Code has been plagiarized from [here](https://www.geeksforgeeks.org/queue-using-stacks/)

**Comment by achalesh27022003** [comment]
What to do then sir to remove plagriarism now? I learnt from there only.

**Comment by achalesh27022003** [comment]
Sir well in that code there is no dynamic input taken, I have taken so. And please guide what to do next

**Comment by HarshCasper** [comment]
> What to do then sir to remove plagriarism now? I learnt from there only.

Learning from a source does not give a right to plagiarize that source. I'm closing the PR and would beseech you to come up with your own code even though you can learn from any resources. Pushing plagiarized code is count as malicious to the overall health of the project.

**Pull request closed without merging**